css/index.css

```diff
@@ -101,17 +125,106 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1 {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
```

Comment:
While creating a brand new class to each box works fine, it isn't best practice. A better approach would be to leave the .box class and target the individual boxes using CSS Selectors such as "nth-child()".